scripts/generator/graphql_generator/csharp/Editor/ShopifyEditorStyleSettings.cs.erb

```diff
@@ -0,0 +1,15 @@
+﻿namespace <%= namespace %>.SDK {
+    using UnityEngine;
+
+    public class ShopifyEditorStyleSettings : ScriptableObject {
```

Comment:
This type is for the asset where I configure the different `GUIStyle`s and colors, etc. I chose this approach because its less brittle to load one master object from the runtime and trust unity's (very good) linking + serialization rather than load all of these things one by one.